### PR TITLE
Rough in how redefining this as just CDNs looks

### PR DESCRIPTION
**Pull request by mnot**

### CODE REVIEW

File: draft-cdn-control-header.md [hunk 1]
```diff
@@ -60,9 +60,9 @@ See also the draft's current status in the IETF datatracker, at
 
 # Introduction
 
-Many HTTP origin servers use gateway caches to speed up distributing their content, either locally (sometimes called a "reverse proxy cache") or in a distributed fashion ("Content Delivery Networks").
+Many HTTP origin servers Content Delivery Networks (i.e., distributed HTTP gateways, usually implementing caches) to speed up distributing their content.
```
Comment (sludin) [inline draft-cdn-control-header.md]: I think you mean to say "server use Content Delivery Networks..."

File: draft-cdn-control-header.md [hunk 2]
```diff
@@ -75,11 +75,11 @@ shown here.
...
 
 It is a Dictionary Structured Header {{!I-D.ietf-httpbis-header-structure}}, whose members can be any directive registered in the HTTP Cache Directive Registry <https://www.iana.org/assignments/http-cache-directives/http-cache-directives.xhtml>.
 
-For gateway caches, when a valid CDN-Cache-Control is present in a response, it MUST take precedence over Cache-Control and Expires headers. In other words, CDN-Cache-Control disables all cache directives in other header fields, and is a wholly separate way to control the cache. Note that this is on a response-by-response basis; if CDN-Cache-Control is not present, caches MAY fall back to other control mechanisms as required by HTTP {{!I-D.ietf-httpbis-cache}}.
+When a valid CDN-Cache-Control is present in a response, CDN caches MUST ignore the Cache-Control and Expires response headers in that response. As such, CDN-Cache-Control is a wholly separate way to control the CDN cache. Note that this is on a response-by-response basis; if CDN-Cache-Control is not present, CDN caches MAY fall back to other control mechanisms as required by HTTP {{!I-D.ietf-httpbis-cache}}.
```
Comment (sludin) [inline draft-cdn-control-header.md]: Should that say "When a valid CDN-Cache-Control header is present"?

**Review by sludin — CHANGES_REQUESTED** [review]

File: draft-cdn-control-header.md (hunk 2 above)
Comment (eaufavor) [reply draft-cdn-control-header.md]: This reminds me of an issue I want to bring up: do we want to support directives across multiple CDN-control-headers?

File: draft-cdn-control-header.md (hunk 1 above)
Comment (mnot) [reply draft-cdn-control-header.md]:
```suggestion
Many HTTP origin servers use Content Delivery Networks (i.e., distributed HTTP gateways, usually implementing caches) to speed up distributing their content.
```

File: draft-cdn-control-header.md (hunk 2 above)
Comment (mnot) [reply draft-cdn-control-header.md]:
```suggestion
When a valid CDN-Cache-Control header field is present in a response, CDN caches MUST ignore the Cache-Control and Expires response headers in that response. As such, CDN-Cache-Control is a wholly separate way to control the CDN cache. Note that this is on a response-by-response basis; if CDN-Cache-Control is not present, CDN caches MAY fall back to other control mechanisms as required by HTTP {{!I-D.ietf-httpbis-cache}}.
```

Comment (mnot) [reply draft-cdn-control-header.md]: @eaufavor it has to - an upstream intermediary might combine two headers (as with any header field, except Set-Cookie).

**Review by eaufavor — APPROVED** [review]

**Pull request merged**